src/js/tabs/pid_tuning.js

```diff
@@ -516,6 +516,11 @@ TABS.pid_tuning.initialize = function (callback) {
             $('.tab-pid_tuning .pidTuningFeatures').hide();
         }
 
+        if (CONFIG.flightControllerIdentifier == 'BTFL' && semver.lt(CONFIG.flightControllerVersion, '3.1.6')
+            || CONFIG.flightControllerIdentifier == 'CLFL'&& semver.lt(CONFIG.flightControllerVersion, '2.0.0')) {
```

Comment:
If I'm not wrong, we are not supporting CF in the configuration. So maybe is better to ignore it to simplify the code and future updates.

Reply:
Sure, I'll remove it. I just looked at existing code to say how people have written `flightControllerVersion` version conditions and saw this.

Reply:
Checking `flightControllerVersion` in the configurator is wrong in any case, you should check `apiVersion`, like so:

https://github.com/betaflight/betaflight-configurator/blob/65fd419649f83384c3d14d5f1976d112806c0a88/src/js/tabs/osd.js#L387

Reply:
True!! I have played too much with the Blackbox lately and I forgot that. Good catch!!

Reply:
Yes, that was my initial approach (since I saw that this is how the majority of the code is written). The problem was that the MW version does not change from 3.1.5 to 3.1.6 - it stays 1.31 (it does not change in 3.1.7 either - only in 3.2.0).
That's why resorted to checking the fw version.

What should be done in such a case?